tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -186,6 +186,21 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
        |""".stripMargin,
   )
 
+  check(
```

Comment:
Could you also add a `checkEdit` test ? We could make sure how the file looks afterwards exactly.